docs/src/Typeahead.doc.js

```diff
@@ -282,6 +288,26 @@ function TypeaheadExample() {
   />,
 );
 
+card(
+  <Example
+    id="disabledExample"
+    name="Disabled"
+    defaultCode={`
+function TypeaheadExample() {
```

Comment:
A nice additional to this example would be a button that controls the disabled state, so the user can play with it and see how that state is controlled programmatically

Reply:
I'd hold off on this until we're doing more complex interactive examples.